README.md

```diff
@@ -8,30 +8,61 @@ This repository holds the ReStructuredText (RST) source, and other files, for us
 
 Follow the documentation to [set up your environment](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#setting-up-your-environment-to-build-documentation-locally) and then [build Ansible community documentation locally](https://docs.ansible.com/ansible/latest/community/documentation_contributions.html#building-the-documentation-locally)
 
-## Using nox
+## Testing your changes
 
-This project includes a `nox` configuration to automate checks and other functions.
-You should use `nox` to run checks locally before you submit a pull request.
+We welcome all contributions to Ansible community documentation.
+If you plan to submit a pull request with changes, you should first run tests from this repository.
+Doing so ensures that your pull request does not trigger any ci failures.
 
-Install `nox` using `python3 -m pip install nox` or your distribution's package manager.
+## Preparing to run tests
 
-Run `nox --list` from the repository root to view available sessions.
+This project includes a `nox` configuration to automate tests, checks, and other functions.
 
-Run `nox` with no arguments to execute the default sessions.
+1. Install `nox` using `python3 -m pip install nox` or your distribution's package manager.
+2. Run `nox --list` from the repository root to view available sessions.
 
-## Running the spelling check
+Each `nox` session creates a temporary environment that installs all requirements and runs the test or check.
+This means you only need to run one command to perform the test accurately and consistently.
 
-This repository uses [`codespell`](https://github.com/codespell-project/codespell) to check for common spelling mistakes in the documentation source.
+## Running all tests
 
-Run `nox -s spelling` to check spelling.
+Run `nox` with no arguments to run all available sessions.
 
-Run `nox -s spelling -- -w` to correct spelling errors.
+## Running the docs-build session
```

Comment:
```suggestion
## Running the docs-build checker
```

At some point, we'll add session to do a regular docs build, so let's specify that this is for the **checker**.